ui/src/kapacitor/components/AlertTabs.js

```diff
@@ -125,12 +126,13 @@ class AlertTabs extends Component {
 
   render() {
     const {configSections} = this.state
-
+    const {locationState} = this.props
     if (!configSections) {
       return null
     }
     const supportedConfigs = {
       alerta: {
+        index: 0,
```

Comment:
is it possible to use something an object method, such as Object.keys, or the key name itself, to infer these indices? it's unusual and a bit of a code smell to define an index per object.

Reply:
I hated putting in index, but couldn't figure out an alternative, thanks for suggesting Object.keys- did this.